docs/en/_sidebar.md

```diff
@@ -17,6 +17,8 @@
         - [Workload Type](/en/cue/workload-type.md)
         - [Trait](/en/cue/trait.md)
         - [Advanced Features](/en/cue/status.md)
+      - HELM
+        - [Chart As Capability](/en/helm/chart-as-capability.md)
 
```

Comment:
Besides the workload and trait section in Helm, please also add a section about [Limitations and known issues] just like https://shipper-k8s.readthedocs.io/en/latest/limitations.html#chart-restrictions